pom.xml

```diff
@@ -162,47 +143,34 @@
         <version>${guava.version}</version>
       </dependency>
       <dependency>
-        <groupId>org.apache.abdera</groupId>
-        <artifactId>abdera-parser</artifactId>
```

Comment:
My basic technique was this: what packages does the artifact export? And are they used in the code anywhere?

Sometimes, there is a transitive dependency conflict and it is necessary to explicitly load a particular version so that the class loader can resolve dependencies properly, but this did not appear to be the case with abdera.